src/Program.cs

```diff
@@ -1,5 +1,58 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿using System;
+using System.Reflection;
+using Arcane.Framework.Contracts;
+using Arcane.Framework.Providers;
+using Arcane.Framework.Providers.Hosting;
+using Arcane.Stream.Cdm;
+using Arcane.Stream.Cdm.Models;
+using Arcane.Stream.Cdm.Models.Base;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+using Snd.Sdk.Logs.Providers;
+using Snd.Sdk.Metrics.Configurations;
+using Snd.Sdk.Metrics.Providers;
+using Snd.Sdk.Storage.Providers;
+using Snd.Sdk.Storage.Providers.Configurations;
 
-using System;
+Log.Logger = DefaultLoggingProvider.CreateBootstrapLogger(nameof(Arcane));
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+int exitCode;
+try
+{
+    exitCode = await Host.CreateDefaultBuilder(args).AddDatadogLogging((_, _, configuration) => configuration.WriteTo.Console())
+        .ConfigureRequiredServices(services =>
+        {
+            return services.AddStreamGraphBuilder<CdmChangeFeedGraphBuilder>(context =>
```

Comment:
This is not needed here, it was implemented in RestApi stream because it has 4 contexts types for one runner. 
Here you have only one context class, so you can do it without reflection. See the [Blob Storage stream](https://github.com/SneaksAndData/arcane-stream-blob-storage/blob/main/src/Program.cs#L37) for reference.